modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -7,26 +7,22 @@ endif::[]
 :imagesdir: ../assets/images
```

Comment:
HIDDEN TEXT above graphic
No edits

= Discover What’s Changed in a Project
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords: